Calculator/Calculator/Form1.cs

```diff
@@ -0,0 +1,199 @@
+using CalculatorLogic;
+
+namespace Calculator
+{
+    public partial class Form1 : Form
```

Comment:
```suggestion
    public partial class CalculatorForm : Form
```
Должно быть говорящее имя у классов. И имена файлов, в котором этот класс описан тоже нужно переименовать